cli/purge.go

```diff
@@ -0,0 +1,38 @@
+package cli
+
+import (
+	"github.com/stelligent/mu/common"
+	"github.com/stelligent/mu/workflows"
+	"github.com/urfave/cli"
+	"os"
+)
+
+func newPurgeCommand(ctx *common.Context) *cli.Command {
+	cmd := &cli.Command{
+		Name:    "purge",
+		Aliases: []string{"nuke"},
+		Usage:   "purge",
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "yes, y",
+				Usage: "suppresses (Y/N) confirmation prompt",
+			},
+			cli.StringFlag{
```

Comment:
this seems redundant since it's already defined at: https://github.com/stelligent/mu/blob/develop/cli/app.go#L117

Reply:
fixed

Reply:
i still see a redundant `namespace` flag